requirements.txt

```diff
@@ -5,6 +5,10 @@ psycopg2==2.9.9 --no-binary=psycopg2
 Django==4.2.16
 libsass==0.23.0
 
+# Optional Trac dependencies that make DeprecationWarning go away
```

Comment:
Do we want to note which deprecation warnings?

Reply:
I've added a comment on how to check if those two dependencies are still needed in the future